release-engineering/role-handbooks/branch-manager.md

```diff
@@ -264,6 +316,8 @@ This presumes reproducible builds and that the CI tests are meaningful relative
 
 ##### krel gcbmgr release
 
+Before running the release step, please refer to the [Image Promotion documentation](https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/release-image-promotion.md).
```

Comment:
I've seen somewhere on Slack that since VDF, it's required to promote alpha images as well, so I added a comment about that to the handbook.

Reply:
Yep :) It now also part of the checklist https://github.com/kubernetes/sig-release/pull/1264